LearningMachine/app/src/main/java/com/learningmachine/android/app/ui/cert/CertificateFragment.java

```diff
@@ -173,7 +166,7 @@ private String displayHTML(BlockCert certificate) {
         String customCss = String.format("body { font-size: 12px; line-height: 1.5; margin:20px; %s } body > section { padding: 0; } body section { max-width: 100%%; word-break: break-word; } body img { max-width: 100%%; height: auto; width: inherit; }", bodyDisplay);
         String autolinkerScript = getAutolinkerScript();
-        return String.format("<!doctype html><html class=\"no-js\" lang=\"\"><head><meta charset=\"utf-8\"><meta http-equiv=\"x-ua-compatible\" content=\"ie=edge\"><title></title><meta content=\"width=device-width, initial-scale=1.0%s\" name=\"viewport\" /><meta name=”viewport” content=”width=device-width” /><style type=\"text/css\">%s</style><style type=\"text/css\">%s</style></head><body><div id=\"_displayHTML\">%s</div></script><script type=\"text/javascript\">%s</script></body></html>", scalingMetaContent, normalizeCss, customCss, displayHTML, autolinkerScript);
+        return String.format("<!doctype html><html class=\"no-js\" lang=\"\"><head><meta charset=\"utf-8\"><meta http-equiv=\"x-ua-compatible\" content=\"ie=edge\"><title></title><meta content=\"width=device-width, initial-scale=1.0%s\" name=\"viewport\" /><meta name=”viewport” content=”width=device-width” /><style type=\"text/css\">%s</style><style type=\"text/css\">%s</style></head><body><div id=\"_displayHTML\">%s</div><script type=\"text/javascript\">%s</script></body></html>", scalingMetaContent, normalizeCss, customCss, displayHTML, autolinkerScript);
```

Comment:
Good catch. If we really wanted to be fully compliant with html5, we should remove the auto-close of the two `meta` tags:
```
<meta content=\"width=device-width, initial-scale=1.0%s\" name=\"viewport\" />
<meta name=”viewport” content=”width=device-width” />
```